examples/llm/src/models/mosaic_gpt.py

```diff
@@ -308,6 +308,19 @@ def __init__(self, om_model_config: DictConfig):
             'Perplexity':
                 Perplexity(),
         }
+        if self.hf_config.loss_fn == 'fused_crossentropy':
+            try:
+                from flash_attn.losses.cross_entropy import CrossEntropyLoss as FusedCrossEntropyLoss  # type: ignore # isort: skip
+                print('Using Fused Cross Entropy Loss.')
+                self.loss_fn = FusedCrossEntropyLoss(inplace_backward=False,
```

Comment:
why `inplace_backward=False`?
(maybe more for my curiosity, less for the PR; I'd expect `True` to use less mem, y set false?)

if `inplace_backward=False` is the default, should we be setting it again?